semgrep/semgrep/target_manager.py

```diff
@@ -319,7 +320,12 @@ def filter_includes(
 
         includes = TargetManager.preprocess_path_patterns(includes)
         return frozenset(
-            wcglob.globfilter(arr, includes, flags=wcglob.GLOBSTAR | wcglob.DOTGLOB)
+            cast(
```

Comment:
maybe a comment here explaining why we need that? What is wrong with wcglob? Or do we need
to update the stub for this library?

Reply:
We don't have control over the stub. I'll add a comment

Reply:
ah lol the auto-merge went through before your comment